src/coreclr/tests/testenvironment.proj

```diff
@@ -65,6 +66,7 @@
     <TestEnvironment>
       <TieredCompilation>0</TieredCompilation>
       <DbgEnableMiniDump Condition="'$(TargetsWindows)' != 'true'">1</DbgEnableMiniDump> <!-- Enable minidumps for all scenarios -->
+      <DbgEnableElfDumpOnMacOS Condition="'$(TargetsWindows)' != 'true'">1</DbgEnableElfDumpOnMacOS> <!-- Enable minidumps for OSX -->
```

Comment:
Should we rather use `Condition="'$(TargetsOSX)' == 'true'"`?

Reply:
Done.